aptos-move/aptos-vm-environment/src/environment.rs

```diff
@@ -165,6 +170,10 @@ struct Environment {
 
     /// Hash of configs used in this environment. Used to be able to compare environments.
     hash: [u8; 32],
+    /// Bytes of serialized verifier config. Used to detect any changes in verification configs.
```

Comment:
Could we add a clear comment/note for the `VerifierConfig` regarding restrictions on how to extend it? Otherwise, with field re-ordering (in fact, I think the first field was changed very recently) etc., we could run into issues where the bytes are the same but they impose differing behavior on the verifier.

Reply:
done